wgs.yaml

```diff
@@ -518,5 +518,56 @@ workinggroups:
   - name: xgboost-operator
     owners:
     - https://raw.githubusercontent.com/kubeflow/xgboost-operator/master/OWNERS
+- dir: wg-data
+  name: WG Data (name provisional)
+  mission_statement: >
+    TODO: WG Data is a provisional name to bootstrap this proposal work.
+    This is being co-located with the WG for Spark scope, as suggested for the Spark proposal.
+    The goal of Spark on Kubernetes Operator is to simplify the capability of running Apache Spark on Kubernetes. It automates deployment and simplifies lifecycle management of Spark Jobs on Kubernetes.
+    The goal of Model Registry is gather, analyze, and develop model registry requirements of Kubeflow community users.
+
+  charter_link: charter.md
+  label: area/wg-data
+  leadership:
+    chairs:
+    - github: andreyvelich
+      name: Andrey Velichkevich
+      company: Apple
+    - github: rimolive
+      name: Ricardo Martinelli de Oliveira
+      company: Red Hat
+    - github: Tomcli
+      name: Tommy Li
+      company: IBM
+    tech_leads:
+    - github: dhirajsb
+      name: Dhiraj Bokde
+      company: Red Hat
+    - github: andreyvelich
+      name: Andrey Velichkevich
+      company: Apple
```

Comment:
@vara-bonthu @yuchaoran2011 @mwielgus being a maintainer of Spark Operator component, do you want to be part of this WG tech leads or chairs ?
It's ok if we are going to have different folks on tech lead and chairs.

Reply:
👋 Would love to be considered as lead or chair for Feast, if that's an option. 😄

Reply:
Makes sense! Please let me know if you need anything from me, I'm happy to discuss it during a call or async as well.